scripts/experiments_images/2stage_knn.jl

```diff
@@ -0,0 +1,133 @@
+using DrWatson
+@quickactivate
+using ArgParse
+using GenerativeAD
+import StatsBase: fit!, predict
+using StatsBase
+using BSON
+# because of vae and 2stage
+using DataFrames
+using CSV
+using ValueHistories
+using Flux
+using ConditionalDists
+using GenerativeModels
+import GenerativeModels: VAE
+using Distributions
+using DistributionsAD
+
+s = ArgParseSettings()
+@add_arg_table! s begin
+   "max_seed"
+        required = true
+        arg_type = Int
+        help = "seed"
+    "dataset"
+        required = true
+        arg_type = String
+        help = "dataset"
+    "tab_name"
+        required = true
+        arg_type = String
+        help = "name of tab -> example: vae_LOSS_images, wae-vamp_AUC_images"
+    "anomaly_classes"
+		arg_type = Int
+		default = 10
+        help = "number of anomaly classes"
+end
+parsed_args = parse_args(ARGS, s)
+@unpack dataset, max_seed, tab_name, anomaly_classes = parsed_args
+
+#######################################################################################
+################ THIS PART IS TO BE PROVIDED FOR EACH MODEL SEPARATELY ################
+
+sp = split(tab_name, "_")
+enc = sp[1]
+criterion = lowercase(sp[2])
+
+modelname = "$(enc)+knn"
```

Comment:
I still dont like the plus, I would go with underscore or -.

Reply:
done